packages/grid/x-data-grid/src/components/cell/GridActionsCellItem.tsx

```diff
@@ -6,13 +6,12 @@ import ListItemIcon from '@mui/material/ListItemIcon';
 import { useGridRootProps } from '../../hooks/utils/useGridRootProps';
 
 export type GridActionsCellItemProps = {
-  label: string;
   icon?: React.ReactElement;
   /** from https://mui.com/material-ui/api/button-base/#ButtonBase-prop-component */
   component?: React.ElementType;
 } & (
-  | ({ showInMenu?: false; icon: React.ReactElement } & IconButtonProps)
-  | ({ showInMenu: true } & MenuItemProps)
+  | ({ showInMenu?: false; label: string; icon: React.ReactElement } & IconButtonProps)
+  | ({ showInMenu: true; label: React.ReactElement } & MenuItemProps)
```

Comment:
is it possible to change this to `ReactNode` instead of `ReactElement`? it should allow `string` and I will be able to revert the fragments
```suggestion
  | ({ showInMenu: true; label: React.ReactNode} & MenuItemProps)
```